rats-processors/test/python/rats_test/pipelines_app/example/_pipeline2.py

```diff
@@ -60,3 +60,11 @@ def p2(self) -> ux.Pipeline[P2Inputs, P2Outputs]:
         p2 = self.get(apps.method_service_id(self.train_model))
         p = self.combine([p1, p2], dependencies=[p1.outputs.data >> p2.inputs.data])
         return cast(ux.Pipeline[P2Inputs, P2Outputs], p)
+
+    @apps.group(rpa.PipelineRegistryGroups.EXECUTABLE_PIPELINES)
```

Comment:
How about we have a decorator `executable_pipeline` that takes a name and doc and is used on top of the pipeline provider method?